### ISSUE TITLE
terraform-docs 0.12.1 SHA256 mis-match

### ISSUE THREAD

**Issue by robinbowes** [Homebrew/homebrew-core#74194]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
$ brew gist-logs terraform-docs
Error: No logs.
```

```
$ brew config
HOMEBREW_VERSION: 3.0.10-51-g1bbed03
ORIGIN: https://github.com/Homebrew/brew
HEAD: 1bbed03c093b0c57920f28f2bcfa8d785b67ddfa
Last commit: 71 minutes ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 7fcbe464d34de7785a1eda5c2026feaf9abf7a00
Core tap last commit: 18 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: ["--appdir=/Applications"]
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit icelake
Clang: 12.0 build 1200
Git: 2.31.1 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.2.3-x86_64
CLT: 12.4.0.0.1.1610135815
Xcode: 12.4
```

```
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Users/robin/.pyenv/shims/python3.6m-config
  /Users/robin/.pyenv/shims/python3.9-config
  /Users/robin/.pyenv/shims/python-config
  /Users/robin/.pyenv/shims/python3-config
  /Users/robin/.pyenv/shims/python3.6-config

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/usr/local/sbin:$PATH"' >> /Users/robin/.bash_profile
```
### 

- [ ] The `brew doctor` above contains no "Warning" lines.

### What were you trying to do (and why)?

I'm trying to upgrade terraform-docs from 0.12.0 to 0.12.1

### What happened (include all command output)?

```
$ brew upgrade terraform-docs
==> Upgrading 1 outdated package:
terraform-docs 0.12.0 -> 0.12.1
==> Upgrading terraform-docs 0.12.0 -> 0.12.1
==> Downloading https://homebrew.bintray.com/bottles/terraform-docs-0.12.1.big_sur.bottle.tar.gz
==> Downloading from https://d29vzk4ow07wi7.cloudfront.net/e2f72b98a3686ca300ccae49189a9a434c71b45b7a00f810d9d1cc5ff3c97af0?response-c
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 191254f25c3dede4fa18022dc3c72b8d4b88be73a007d52dcb0c1bc700f2f031
  Actual: e2f72b98a3686ca300ccae49189a9a434c71b45b7a00f810d9d1cc5ff3c97af0
    File: /Users/robin/Library/Caches/Homebrew/downloads/67489974e07eb3d894cb1348ab47dd17cfedd1b14cac3a24282eefc14958f428--terraform-docs-0.12.1.big_sur.bottle.tar.gz
To retry an incomplete download, remove the file above.
```

### What did you expect to happen?

I expect terraform-docs 0.12.1 to install cleanly

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew upgrade terraform-docs
```

**Comment by carlocab** [comment]
There's either something wrong with your Internet connection, or your bottle download was corrupted. [1] Do
```
rm -f /Users/robin/Library/Caches/Homebrew/downloads/67489974e07eb3d894cb1348ab47dd17cfedd1b14cac3a24282eefc14958f428--terraform-docs-0.12.1.big_sur.bottle.tar.gz
```
then try again.

[1] On my machine:
```
❯ curl -fsSL https://homebrew.bintray.com/bottles/terraform-docs-0.12.1.big_sur.bottle.tar.gz | sha256sum
e2f72b98a3686ca300ccae49189a9a434c71b45b7a00f810d9d1cc5ff3c97af0  -
```
This matches the expected checksum, so there should be nothing wrong with the actual bottle on Bintray.

**Comment by robinbowes** [comment]
Hmm, I tried removing that. It downloaded just the same. Let me try again...
```
$ rm -f /Users/robin/Library/Caches/Homebrew/downloads/67489974e07eb3d894cb1348ab47dd17cfedd1b14cac3a24282eefc14958f428--terraform-docs-0.12.1.big_sur.bottle.tar.gz
```

```
$ curl -fsSL https://homebrew.bintray.com/bottles/terraform-docs-0.12.1.big_sur.bottle.tar.gz | sha256sum
e2f72b98a3686ca300ccae49189a9a434c71b45b7a00f810d9d1cc5ff3c97af0  -
```
That's not the expected checksum, btw.

```
$ brew upgrade terraform-docs
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 1 formula.

==> Upgrading 1 outdated package:
terraform-docs 0.12.0 -> 0.12.1
==> Upgrading terraform-docs 0.12.0 -> 0.12.1
==> Downloading https://homebrew.bintray.com/bottles/terraform-docs-0.12.1.big_sur.bottle.tar.gz
==> Downloading from https://d29vzk4ow07wi7.cloudfront.net/e2f72b98a3686ca300ccae49189a9a434c71b45b7a00f810d9d1cc5ff3c97af0?response-c
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 191254f25c3dede4fa18022dc3c72b8d4b88be73a007d52dcb0c1bc700f2f031
  Actual: e2f72b98a3686ca300ccae49189a9a434c71b45b7a00f810d9d1cc5ff3c97af0
    File: /Users/robin/Library/Caches/Homebrew/downloads/67489974e07eb3d894cb1348ab47dd17cfedd1b14cac3a24282eefc14958f428--terraform-docs-0.12.1.big_sur.bottle.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by robinbowes** [comment]
https://github.com/Homebrew/homebrew-core/blob/master/Formula/terraform-docs.rb#L10

**Comment by carlocab** [comment]
You're right; I had misread the `Expected:` and `Actual:` output in your first post. Let me look into it.

**Comment by carlocab** [comment]
Ok, I see the problem.

The bottle hash in the formula is the hash from the bottles built at #74171. However, the Big Sur bottle on Bintray is the bottle built at #74141.

If you want to get around the mismatch for now, just change the checksum in the formula to match the one that's being downloaded.

Apologies for the inconvenience; it seems we need to be more careful about duplicate PRs here. Normally they wouldn't cause any problems, except in the instance when @BrewTestBot tries to merge them simultaneously (as what happened with #74141 and #74171).

CC @Homebrew/core

**Comment by khos2ow** [comment]
Maintainer of @terraform-docs here :wave:! Is this something we should've done or a mistake we made along the way or just a matter of duplicated PR on homebrew side?

**Comment by carlocab** [comment]
@khos2ow no, you're all good here. This was a problem on our end. Thanks for checking in, though!
